tests/importance_tests/test_fanova.py

```diff
@@ -55,3 +55,20 @@ def test_fanova_importance_evaluator_seed() -> None:
     evaluator = FanovaImportanceEvaluator(seed=3)
     param_importance_different_seed = evaluator.evaluate(study)
     assert param_importance != param_importance_different_seed
+
+
+def test_fanova_importance_evaluator_with_target() -> None:
+    # Assumes that `seed` can be fixed to reproduce identical results.
+
+    study = create_study(sampler=RandomSampler(seed=0))
+    study.optimize(objective, n_trials=100)
```

Comment:
`n_trials = 100` is too big for the unit test. How about setting `n_trials = 3` which is the same as above to reduce the execution time?